### DISCUSSION TITLE
现在绳网的氛围好舒服，要是一直这样就好了

### DISCUSSION THREAD

**Discussion by 0build0** [share121/inter-knot#3136] [其他]
<img src="https://img-static.mihoyo.com/communityweb/upload/66f5c4854e9212e50fae7aeb97995a73.png" alt="三月七_开心" width="75" >希望绳网的这种氛围一直保持下去，不要被不明生物搞坏🙏

**Comment by qqqqqq77777** [comment]
赞同艾<img src="https://img-static.mihoyo.com/communityweb/upload/9f49c04a7aeb5324760df769e1d9acbe.png" alt="希露瓦 放电" width="75" >

**Comment by RaphaelHider** [comment]
有东西来破坏我第一个当将军

**Reply by Nie7bai** [reply]
直接踩踏，全给踩死了

**Comment by Hod-141** [comment]
同意<img src="https://upload-bbs.miyoushe.com/upload/2023/02/03/a16450f92757f9d45588a8541fbbab09_4901912883842187271.gif" alt="米游姬-感谢" width="75" >

**Comment by Lyou777** [comment]
赛博空间最后的净土
![26080e61a61855b46fd992dcc8cb44d2_3620209287005874272](https://github.com/user-attachments/assets/e9d1d658-8216-4a2b-b69f-c7ce153de833)

**Comment by warning-Kirov-reporting** [comment]
抗性拉满，当乐子人<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by 00zzxx** [comment]
赞同——————

**Comment by Xy9223** [comment]
人少+门槛高，所以人均素质会好一些<img src="https://upload-bbs.mihoyo.com/upload/2022/09/09/39ed096de3b1c59e09a9bfffaefcf946_6861543172227748935.png" alt="派蒙-伪装" width="75" >
